bin/run-migration-job.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+This script will either migrate or rollback the swoop database to a specified
+migration version. The migration version, action type (of either migrate/rollback),
+and an override parameter are all specified in the values.yml from the Swoop-DB helm
+chart.
+
+It requires the following environment variables be set:
+
+    - PGHOST:
+          Postgres K8s service name which will be used to connect to the database
+    - PGUSER:
+          name of the user (role) that will perform the migrations
+    - PGPORT:
+          port number of the database container
+    - PGDATABASE: name of the database which will be used for migrations
+    - Any additional libpq-supported connection parameters
+          (https://www.postgresql.org/docs/current/libpq-envars.html)
+"""
+import argparse
+import asyncio
+import os
+import sys
+
+from buildpg import V, funcs, render
+
+from swoop.db import SwoopDB
+
+APPLICATION_ROLES = ["user_api", "user_caboose", "user_conductor"]
+
+
+def stderr(*args, **kwargs) -> None:
+    kwargs["file"] = sys.stderr
+    print(*args, **kwargs)
+
+
+def check_positive(value) -> int:
+    ivalue = int(value)
+    if ivalue <= 0:
+        raise argparse.ArgumentTypeError(
+            "%s is an invalid migration version number. Only \
+                positive values are supported."
+            % value
+        )
+    return ivalue
+
+
+async def run_migrations() -> None:
+    dbname: str = os.environ["PGDATABASE"]
+
+    parser = argparse.ArgumentParser()
```

Comment:
Forgot to say on my review, it feels a little weird to be using argparse here and not supporting env vars for these parameters. Everything else we have supports env vars for options. For consistency, you could:

* Not use argparse and simply parse env vars
* Look at using the `EnvDefault` action from `dbami`: https://github.com/Element84/dbami/blob/main/src/dbami/cli.py#L148 -- except this conflicts with the `store_true`/`store_false` actions, so I think you would need to set the type of flags to use a function that evaluates the truthiness of the supplied value and coerces it into a boolean.

I guess you would want said function to coerce to boolean regardless, and I think the stdlib even has one??? <quick search...> Nope, it's from distutils, which has been deprecated: https://stackoverflow.com/a/18472142. The snippet there looks like a good starting point though.